topologies/binding/binding.go

```diff
@@ -165,6 +168,30 @@ func (d *staticDUT) DialCLI(ctx context.Context, opts ...grpc.DialOption) (bindi
 	return newCLI(sc)
 }
 
+func (a *staticATE) DialGNMI(ctx context.Context, opts ...grpc.DialOption) (gpb.GNMIClient, error) {
+	if a.dev.Gnmi == nil {
```

Comment:
If you look at the staticDUT version, it uses a resolver which can infer the endpoint from the dut hostname and the default gNMI port, so the `Gnmi.Target` is optional.

So if ixia-c-grpc and ixia-c-gnmi have a preferred default port, we could also add similar logic to the resolver so the `Gnmi.Target` and `Otg.Target` can be omitted. WDYT?

Reply:
I agree on the `Gnmi.Target` part but according to me `Otg` option need to be there in the binding in order to provide the demarcation between `IxNetwork` and `Otg`. 
I have add this error if user configure both `Otg` as well as `IxNetwork` at the ATE binding file. We need this validation upfront as `ixSession` execute within `reserveIxSessions` .
```go
if ate.Otg != nil && ate.Ixnetwork != nil {
    return nil, fmt.Errorf("otg and ixnetwork are not mutually exclusive. Please configure one of them in ate %s binding.", ate.Name)
}
```
I have changed it accordingly. Please review.

Reply:
I'm fine to keep both Gnmi and Otg. I'm just suggesting that the Target field could be optional if both the ixia-c-gnmi and ixia-c-grpc can infer the endpoint address from the ate.Hostname and default ports.

By the way I think you meant to write that "otg and ixnetwork **are** mutually exclusive" rather than "are **not** mutually exclusive?"

Reply:
I have defined `ixia-c-grpc` and `ixia-c-gnmi` default ports
```go
	ateGnmiPort = flag.Int("ate_gnmi_port", 50051, "default ATE gNMI port")
	ateOtgPort  = flag.Int("ate_grpc_port", 40051, "default ATE gRPC port for running OTG test")
``` 
Also fix the error message